packages/core/types/src/configuration.ts

```diff
@@ -77,7 +77,7 @@ export type FlagsConfig = {
   changePassword?: boolean;
 };
 
-export type PackageManagers = 'pnpm' | 'yarn' | 'npm';
+export type PackageManagers = 'pnpm' | 'yarn' | 'npm' | string;
```

Comment:
Probably there is no much todo if someone uses a different package manger, but I think UI won't render anything.

Reply:
but you can use a theme plugin 😃